source/includes/stackpath/_workloads.md

```diff
@@ -560,3 +560,32 @@ Attributes | &nbsp;
 ------- | -----------
 `taskId` <br/>*string* | The task id related to the workload deletion.
 `taskStatus` <br/>*string* | The status of the operation.
+
+#### Edit workload settings
+
+Change a workload specific settings in a given [environment](#administration-environments). At the moment, only `isRemoteManagementEnabled` is supported for this operation. Any other attributes will be ignored.
+
+```shell
+curl -X PATCH \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/workloads/1b932678-1038-4ab4-9fa4-c4c06e696e20"
+```
+
+> Request body example for a VM Workload type:
```

Comment:
Is it the same attribute for a Container workload? The wording here seems to indicate that it would be different

Reply:
@pdube Yes it's the same attribute for both. Will adjust this accordingly. Standby.

Reply:
Fixed to have a generic message like other places in the documentation.